### ISSUE TITLE
Bioconda Bot is not able to fetch artifacts/packages, and getting krakenuniq as part of the build

### ISSUE THREAD

**Issue by nh13** [bioconda/bioconda-recipes#41153]
See PRs:

* https://github.com/bioconda/bioconda-recipes/pull/40950#issuecomment-1561677902
* https://github.com/bioconda/bioconda-recipes/pull/41130
* https://github.com/bioconda/bioconda-recipes/pull/41146

**Comment by dpryan79** [comment]
Yes, since the CI switch the bot needs to be rewritten (it's looking for artifacts in Azure). I suspect that kraken-uniq is somehow getting stored in the CI cache. It can be ignored.
